Studiosity/EmberSockets
# Replace deprecated fmt call with ES6 template strings

How to test:

* Update `Bowerfile` in eureka to reference this branch for the `ember-sockets` asset. 
* Delete the existing `ember-sockets` asset folder. i.e. `vendor/assets/bower_components/ember-sockets`
* `bundle exec rake bower:install`
* See that you can still have a session.. but bonus.. no more "Ember.String.fmt is deprecated" warnings on the console..
